### PR TITLE
Reformat section "Alternatives to Rstudio"

### PR DESCRIPTION
**Pull request by frederic-santos**
I propose various changes for this section:

- Add Emacs+ESS, Nvim-R and RKWard as alternatives to Rstudio
- Each alternative has now its own URL
- Alternatives are now listed by type (either text editor or advanced IDE) rather than operating system, which avoids duplicates

### CODE REVIEW

**Comment by alexd106** [comment]
Thanks for taking the time to propose these changes Frederic. They look very sensible to me and I'll merge your changes and acknowledge your contribution in the book.

**Comment by alexd106** [comment]
I would suggest changing:
_'A light yet efficient way to write R scripts consists in using advanced text editors such as:'_ to _'A light yet efficient way to write your R scripts using advanced text editors such as:'_

and 

_'Those environments are more powerful than simple text editors, and are similar to Rstudio:'_ to _'**These** environments are more powerful than simple text editors, and are similar to **RStudio**:'_

File: 01-intro.Rmd [hunk 1]
```diff
@@ -202,30 +202,26 @@ The Environment / History / Connections window shows you lots of useful informat
...
-- [Sublime Text][sublime]
-- gedit (comes with Ubuntu already)
-- [Atom][atom]
-- vim/nano (Console based editors)
-
-Which one you choose is up to you. One of the great things about R is that *YOU* get to choose how you want to use R
+Although RStudio is becoming increasingly popular it might not be the best choice for everyone and you certainly don't have to use it to use R effectively. Rather than using an 'all in one' IDE many people choose to use R and a separate script editor to write and execute R code. If you're not familiar with what a script editor is, you can think of it as a bit like a word processor but specifically designed for writing code. Happily, there are many script editors freely available so feel free to download and experiment until you find one you like. Some script editors are only available for certain operating systems and not all are specific to R. Suggestions for script editors are provided below. Which one you choose is up to you: one of the great things about R is that *YOU* get to choose how you want to use R.
+
+### Advanced text editors
+A light yet efficient way to write R scripts consists in using advanced text editors such as:
```
Comment (alexd106) [inline 01-intro.Rmd:208]: I would suggest changing to: 'A light yet efficient way to write your R scripts using advanced text editors such as:'

File: 01-intro.Rmd [hunk 2]
```diff
@@ -202,30 +202,26 @@ The Environment / History / Connections window shows you lots of useful informat
...
+- [Atom][atom] (all operating systems)
+- [BBedit][BBedit] (Mac OS)
+- [gedit][gedit] (Linux; comes with most Linux distributions)
+- [MacVim][macvim] (Mac OS)
+- [Nano][nano] (Linux)
+- [Notepad++][notepad] (Windows)
+- [Sublime Text][sublime] (all operating systems)
+
+### Integrated developement environements
+Those environments are more powerful than simple text editors, and are similar to Rstudio:
```
Comment (alexd106) [inline 01-intro.Rmd:219]: suggest changing to 'These environments are more powerful than simple text editors, and are similar to RStudio:'

**Review by alexd106 — CHANGES_REQUESTED** [review]

**Comment by frederic-santos** [comment]
Thanks, I added your changes in my commit 77a376e.
(And many thanks for your whole book, which is the best online resource I know for beginners!)

**Pull request merged**